packages/generator-one-app-module/generators/app/templates/root-module/src/csp.js

```diff
@@ -1,6 +1,12 @@
 import contentSecurityPolicyBuilder from 'content-security-policy-builder';
 import ip from 'ip';
 
+const developmentAdditions = process.env.NODE_ENV === 'development' ? [
```

Comment:
Instead of doing this here I think we will want to/need to have One App provide these development additions so that modules don't have to worry about it.

Otherwise what I am seeing happening is that when locally trying to use a root module that is not being served locally the CSP does not include these additions and so the module never loads.

Reply:
fixed in ed4f434